test/test_distributed.py

```diff
@@ -358,11 +381,13 @@ def test_reduce_sum(self):
 
     @unittest.skipIf(BACKEND != 'nccl', "Only Nccl supports CUDA reduce")
     @skip_if_no_cuda_distributed
+    @skip_if_no_multigpu
```

Comment:
The `multigpu` name is confusing, because it doesn't require that you have multiple GPUs in this process, but that this process has a unique GPU. Can you please rename it to `skip_if_no_gpu`?